modules/aws_base/tf_module/log_bucket.tf

```diff
@@ -23,9 +23,21 @@ resource "aws_s3_bucket_public_access_block" "log_bucket" {
 
 }
 
+resource "aws_s3_bucket_ownership_controls" "log_bucket" {
+  bucket = aws_s3_bucket.log_bucket.id
+
+  rule {
+    object_ownership = "BucketOwnerPreferred"
```

Comment:
Should we be using [`ObjectWriter`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_ownership_controls#ObjectWriter) instead? This was the default prior to the change. This might be the safer change to achieve parity with older buckets, and not cause a functional diff in existing deployments. However, `BucketOwnerPreferred` might be preferable for new buckets.

Reply:
I agree all around here.  `ObjectWriter` was the old default behavior as far as I remember, and `BucketOwnerPreferred` seems like a good default.

Because we are not doing any cross-account reads/writes out of these log buckets, I think it is pretty safe to change this.  My only concern would be _who_ actually owns the bucket.  Is it the root IAM principal (`arn:aws:iam::{account}:root`) or is it potentially some other principal, like the role that created it?